Tasks/XcodeV5/xcodeutils.ts

```diff
@@ -205,6 +205,19 @@ export function uploadLogFile(logFile: string) {
     }
 }
 
+export function publishXcodeTelemetry(telemetryData: { [key: string]: any; }) {
```

Comment:
I think this would be good as `function publishTelemetry(area: string, feature: string, data: { [key: string]: any; }): void`.  That would be a generic way to do it that could easily be reused for other tasks -- or moved to task-lib.